frontend/src/lib/utils/query/queryUrl.ts

```diff
@@ -8,3 +8,24 @@ export const queryUrl = async <T>(url: string): Promise<T> => {
 
   return response.json();
 };
+
+export const thorWalletQueryUrl = async <T, R = undefined>(
```

Comment:
`lib` is a project agnostic place, when we'll have a project moved to a monorepo it would be an independent package, so it shouldn't have any mention of "Vulti"